stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1947,10 +1591,7 @@ Operation* ReduceWindowOp::getReductionOp(int resultIndex) {
 // We intend to verify the following properties
```

Comment:
Looks like `ReduceWindowOp::getReductionOp` can be deleted now?

Reply:
Seems it is used nowhere in StableHLO and MHLO, deleted now.